utilities/cert_utility.mak

```diff
@@ -16,9 +16,12 @@ endif
 #ifndef GOOGLE_INCLUDE
 #GOOGLE_INCLUDE=/usr/local/include/g
 #endif
+
+# Allows user to over-ride libs path externally depending on machine's install
 ifndef LOCAL_LIB
 LOCAL_LIB=/usr/local/lib
 endif
+
```

Comment:
On `sgx` machine I tested, you have to run this as: `LOCAL_LIB=/usr/local/lib64 make -f cert_utility.mak`